salesagility/SuiteCRM
# Feature/gdpr send confirm opt in email

## Description



The change send an email template to the confirm opt in.

## Motivation and Context

- GDPR

## How To Test This

- Please install suitecrm when testing. 
- Set up email 
- Set up email settings to confirm opt in, auto send email, and select confirm opt in template
- Go to contacts, you should see the tick in the list view, and detail view next to each email address.
- Click an email address
- Send an email
- You should receive a confirm opt in email
- After confirming email refresh contact
- Tick status should have changed.

## Types of changes


### Final checklist
